### DISCUSSION TITLE
Permission Denied Error in Tomcat After Enabling Security

### DISCUSSION THREAD

**Discussion by shobhit2323** [community/community#122542] [Programming Help]
I'm encountering a permission denied error in Tomcat after enabling security and ensuring that all necessary permissions are configured. Here's the error message I'm getting:
**access: access denied ("java.util.PropertyPermission" "org.jboss.remoting3.remote.message.ack.timeout" "read")
java.lang.Exception: Stack trace
	at java.base/java.lang.Thread.dumpStack(Thread.java:1383)
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:462)
**

I've verified that the required permissions are correctly set in my Tomcat security policy file, and I've also confirmed that the property "org.jboss.remoting3.remote.message.ack.timeout" is properly configured in my Tomcat configuration files. Additionally, I've debugged the security settings using -Djava.security.debug=access,failure.

Despite these efforts, I'm still encountering this error. I've checked my code for any potential issues related to accessing this property, and I'm not seeing any obvious problems there either.

Could someone help me understand what might be causing this error and how I can resolve it? Any insights or suggestions would be greatly appreciated. Thank you!

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @shobhit2323 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Reply by shobhit2323** [reply]
thank you
